### DISCUSSION TITLE
MouseEvent.button returns 1 on back/forward button press

### DISCUSSION THREAD

**Discussion by kybin** [wailsapp/wails#1559] [Q&A] [answered]
I'm not sure whether this is wails problem but I need help. :)

I need 3 (back) or 4 (forward) button event for navigation on the app, but only got 0 and 1.

Is there a setting for it? 

I'm using m1 mac currently, if that matters.

Thanks!

**Comment by leaanthony** [comment, marked as answer]
It's best to open tickets for this and list what you've tried

**Reply by kybin** [reply]
Created #1560

Thanks!
